public/r2/v1/swagger/swagger.json

```diff
@@ -184,6 +184,72 @@
                 }
             }
         },
+        "/namespaces/{namespaceID}/ruleset/bulk": {
+            "put": {
+                "tags":[
+                    "namespaces"
+                ],
+                "summary": "Bulk update endpoint for namespace changes",
+                "operationId": "bulkUpdateRuleSet",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "in": "path",
+                        "name": "namespaceID",
+                        "description": "The name of the namespace",
+                        "type": "string",
+                        "required": true
+                    },
+                    {
+                        "in": "body",
+                        "name": "ruleset-change",
```

Comment:
This captures the set of rule changes yeah? If so might be better to call this `ruleset-changes`?